packages/angular/src/generators/application/application.spec.ts

```diff
@@ -1242,6 +1242,16 @@ describe('app', () => {
         ]
       `);
     });
+    it('should generate a correct build target for --bundler=rspack', async () => {
```

Comment:
nit: test case title seems mileading, we're actually validating that no build target should be generated and the config is correctly generated.